tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bottom_bar.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+class BottomBar(Base):
+    _remove_locator = (By.ID, 'remove')
+    _done_locator = (By.ID, 'done')
+
```

Comment:
Ok, thanks, I got that number this way. But adding the __init__makes the test fail giving timeout because it waits to the element to be displayed, but then it stays there and it is not possible to tap on the remove button

def __init__(self, marionette):
        Base.__init__(self, marionette)
        view = self.marionette.find_element(*self._bottom_bar_locator)
        bottom_bar_position = view.rect
        Wait(self.marionette).until(lambda m: view.location['y'] == (bottom_bar_position["y"]))